@xen-orchestra/lite/src/views/new-vm/NewVmView.vue

```diff
@@ -0,0 +1,989 @@
+import FormSelect from '@/components/form/FormSelect.vue'
+import TitleBar from '@/components/TitleBar.vue'
+
+// XenAPI Store imports
+import type { XenApiNetwork, XenApiSr, XenApiVbd, XenApiVdi, XenApiVif, XenApiVm } from '@/libs/xen-api/xen-api.types'
+import { useNetworkStore } from '@/stores/xen-api/network.store'
+import { usePifStore } from '@/stores/xen-api/pif.store'
+import { usePoolStore } from '@/stores/xen-api/pool.store'
+import { useSrStore } from '@/stores/xen-api/sr.store'
+import { useVbdStore } from '@/stores/xen-api/vbd.store'
+import { useVdiStore } from '@/stores/xen-api/vdi.store'
+import { useVifStore } from '@/stores/xen-api/vif.store'
+import { useVmStore } from '@/stores/xen-api/vm.store'
+import { useXenApiStore } from '@/stores/xen-api.store'
+import { type Disk, type NetworkInterface, type VmState } from '@/types/new-vm'
+
+// UI components imports from web-core
+import VtsIcon from '@core/components/icon/VtsIcon.vue'
+import VtsResource from '@core/components/resources/VtsResource.vue'
+import VtsResources from '@core/components/resources/VtsResources.vue'
+import VtsTable from '@core/components/table/VtsTable.vue'
+import UiButton from '@core/components/ui/button/UiButton.vue'
+import UiButtonIcon from '@core/components/ui/button-icon/UiButtonIcon.vue'
+import UiCard from '@core/components/ui/card/UiCard.vue'
+import UiCheckbox from '@core/components/ui/checkbox/UiCheckbox.vue'
+import UiCheckboxGroup from '@core/components/ui/checkbox-group/UiCheckboxGroup.vue'
+import UiInput from '@core/components/ui/input/UiInput.vue'
+import UiLabel from '@core/components/ui/label/UiLabel.vue'
+import UiRadioButton from '@core/components/ui/radio-button/UiRadioButton.vue'
+import UiTextarea from '@core/components/ui/text-area/UiTextarea.vue'
+import UiTitle from '@core/components/ui/title/UiTitle.vue'
+import { vTooltip } from '@core/directives/tooltip.directive'
+
+// Icon Imports
+import {
+  faAlignLeft,
+  faAt,
+  faDatabase,
+  faDisplay,
+  faMemory,
+  faMicrochip,
+  faNetworkWired,
+  faPlus,
+  faTrash,
+} from '@fortawesome/free-solid-svg-icons'
+
+// Defer import
+// Vue imports
+import { computed, reactive, ref } from 'vue'
+import { useI18n } from 'vue-i18n'
+import { useRouter } from 'vue-router'
+
+// Store subscriptions
+const { templates } = useVmStore().subscribe()
+const { pool } = usePoolStore().subscribe()
+const { records: srs, VDIsGroupedBySrName, getByOpaqueRef: getOpaqueRefSr } = useSrStore().subscribe()
+const { records: networks, getByOpaqueRef: getOpaqueRefNetwork } = useNetworkStore().subscribe()
+const { getByOpaqueRef: getOpaqueRefVbd } = useVbdStore().subscribe()
+const { getByOpaqueRef: getOpaqueRefVdi } = useVdiStore().subscribe()
+const { getByOpaqueRef: getOpaqueRefVif } = useVifStore().subscribe()
+const { getByOpaqueRef: getOpaqueRefPif } = usePifStore().subscribe()
+
+// i18n setup
+const { t } = useI18n()
+const router = useRouter()
+
+const isLoading = ref<boolean>(false)
```

Comment:
```suggestion
const isBusy = ref<boolean>(false)
```
There is also a `useBusy` composable that can be used, I think.